File: Carthage/Checkouts/onepassword-extension/Demos/App Demo for iOS/App Demo for iOS/ThankYouViewController.m
//
//  ThankYouViewController.m
//  App Demo for iOS
//
//  Created by Rad Azzouz on 2014-07-21.
//  Copyright (c) 2014 AgileBits. All rights reserved.
//

#import "ThankYouViewController.h"

@interface ThankYouViewController ()

@end

@implementation ThankYouViewController

- (void)viewDidLoad {
	[super viewDidLoad];

	[self.view setBackgroundColor:[[UIColor alloc] initWithPatternImage:[UIImage imageNamed:@"login-background.png"]]];
}

-(UIStatusBarStyle)preferredStatusBarStyle{
	return UIStatusBarStyleLightContent;
}

@end

File: Carthage/Checkouts/onepassword-extension/Demos/App Demo for iOS Swift/App Demo for iOS Swift/ThankYouViewController.swift
//
//  ThankYouViewController.swift
//  App Demo for iOS Swift
//
//  Created by Rad Azzouz on 2015-05-14.
//  Copyright (c) 2015 Agilebits. All rights reserved.
//

import Foundation

class ThankYouViewController: UIViewController {
	override func viewDidLoad() {
		super.viewDidLoad()
		
		if let patternImage = UIImage(named: "login-background.png") {
			self.view.backgroundColor = UIColor(patternImage: patternImage)
		}

	}
	
	override func preferredStatusBarStyle() -> UIStatusBarStyle {
		return UIStatusBarStyle.LightContent
	}
}
